MMM-pages.js

```diff
@@ -69,7 +69,7 @@ Module.register('MMM-pages', {
     this.config.rotationHomePage = Math.max(this.config.rotationHomePage, 0);
 
     if (!this.config.useLockString) {
-      Log.log('[Pages]: User opted to not use lock strings!');
+      Log.info('[MMM-pages] User opted to not use lock strings!');
```

Comment:
nit: Any reason for changing this?

Reply:
Since setting `useLockString` to false, I thought it wouldn't be a bad idea to highlight this a bit in the console using `Log.info`. This was more of a spontaneous intuition than following any real strategy. Now thinking about it, I think the highlighting is unnecessary and have just removed it.
Thanks for the nitpicking!